src/MobileNavbar/MobileNavbar.scss

```diff
@@ -41,7 +41,7 @@
     position: fixed;
     top: 6px;
     right: 6px;
-    z-index: 3;
+    z-index: 4;
```

Comment:
We should probably move all of these z-indexes into a separate file to track them, where we to carry on with the project past today I'd probably ask for this!